apps/client/src/utils/index.js

```diff
@@ -2,6 +2,11 @@ import isBoolean from "lodash.isboolean";
 import isNumber from "lodash.isnumber";
 import isString from "lodash.isstring";
 
+import { topics } from "../config";
+
+// Find a topic by the slug
+export const findTopicBySlug = (slug) => topics.find((t) => t.slug === slug);
```

Comment:
Small test for this one would be very nice.

Reply:
It has a test, but it is on the [Conclusion PR](https://github.com/Amsterdam/vergunningcheck/pull/522/files). This is a PR to develop